test-requirements.txt

```diff
@@ -5,4 +5,5 @@ pytest-xdist==2.1.0
 boto3>=1.10.45
 moto[all]>=1.3.14
 smart-open>=5.0.0
-pyarrow>=2.0.0
\ No newline at end of file
+pyarrow>=2.0.0
+mock==4.0.3
```

Comment:
Not sure why this didn't come up sooner, but without adding `mock` to the test requirements we started getting `ModuleNotFoundError No module named mock`